website/client/components/tasks/taskModal.vue

```diff
@@ -988,6 +988,11 @@ export default {
     focusInput () {
       this.$refs.inputToFocus.focus();
     },
+    handleClick (e) {
+      if (this.$refs.popup && !this.$refs.popup.$el.contains(e.target)) {
```

Comment:
like in #10547 wouldn't be a `showTagsSelect` enough to check if its opened ?

Reply:
in this case we want to check if the click was inside or outside the popup not only if it was open so this looks good